### PR TITLE
modify js and related code

### PR DESCRIPTION
**Pull request by taisuou**
task board
https://github.com/vdaas/web/projects/1#card-39481653

toc
- fix toc (#current ) css to position fixed as default
- add js to open and close toc menu when viewing page on smartphone

page sidebar
-changed arrow color on darktheme

bug fix
-wrong js code worked to add `body` to take some margin. deleted this code

toc
#20 toc
 
page sidebar
#20 page sidebar

### CODE REVIEW

File: themes/vald/layouts/partials/sidebar.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <!-- TODO: attach class "open" nav -->
 <aside class="page">
   {{ $current := .Page.Permalink }}
-  <nav class="open">
+  <nav>
```
Comment (vankichi) [inline themes/vald/layouts/partials/sidebar.html:5]:
```suggestion
  <nav class="open">
```

we should keep attach `open` class before implementation.

File: themes/vald/static/js/script.js [hunk 2]
```diff
@@ -26,10 +26,29 @@
...
+    tocWrap.classList.add('open');
+  }
+
+  function tocClose() {
+    tocWrap.classList.remove('open');
+  }
+
 
   //smooth scroll
   let headerHight = document.getElementById('header').offsetHeight;
```
Comment (vankichi) [inline themes/vald/static/js/script.js:51]:
```suggestion
  const headerHight = document.getElementById('header').offsetHeight;
```

**Comment by vankichi** [comment]
@taisuou please rebase your branch

File: themes/vald/layouts/partials/sidebar.html (hunk 1 above)
Comment (taisuou) [reply themes/vald/layouts/partials/sidebar.html:5]: I changed here in this commit
https://github.com/vdaas/web/pull/21/commits/057e001a95ad796a5c1c5eb104fc8c6d2d5f0cbe

File: themes/vald/static/js/script.js (hunk 2 above)
Comment (taisuou) [reply themes/vald/static/js/script.js]: fixed
https://github.com/vdaas/web/pull/21/commits/9e372d9b75dc652816dad8d436e00b5898c3f016

**Pull request merged**